trade-finance-manager-ui/server/api.js

```diff
@@ -1300,6 +1300,27 @@ const deleteDealCancellation = async (dealId, userToken) => {
   }
 };
 
+/**
+ * Submits the deal cancellation object on a TFM MIN or AIN deal
+ * @param {string} dealId - The deal ID
+ * @param {import('@ukef/dtfs2-common').TfmDealCancellation} cancellation - The deal cancellation object
```

Comment:
Out of interest, if no reason was submitted, would this in fact be a `Partial` of this type instead? (looks like the zod schema it's referencing doesn't have it as optional - but not sure if an empty string would still satisfy the `string` requirement)

Reply:
I tested this a while back, if you submit an empty text input it sends an empty string so the reason should always be defined